app/styles/courses/teacher-class-view.sass

```diff
@@ -0,0 +1,250 @@
+@import "app/styles/bootstrap/variables"
+@import "app/styles/mixins"
+@import "app/styles/style-flat"
+
+$grey-light: #f5f5f5
+$grey-medium: #ebebeb
+$grey-dark: #c9c9c9
+$grey-darker: #9b9b9b
+
+.alternating-background:nth-child(2n+1)
+  background-color: $grey-medium
+
+.alternating-background:nth-child(2n)
+  background-color: $grey-light
+
+#teacher-class-view
+
+  ul.nav-tabs
+    border: none
+    .tab-spacer
+      float: left
+      width: 15px
+      height: 47px
+      border-bottom: thin solid #979797
+      
+    .tab-filler
+      // Triggers block formatting context
+      // http://stackoverflow.com/questions/1260122/expand-a-div-to-take-the-remaining-width
+      overflow: hidden
+      height: 47px
+      border-bottom: thin solid #979797
```

Comment:
Are these extra tab classes necessary? If possible, tabs and bootstrap components in general shouldn't diverge from the documentation. If they don't look exactly like the mockup, that's fine, it's better than having to remember how our tabs are different in the future when they most likely will be used again in the flat style.